source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -0,0 +1,73 @@
+title: Create an AWS IAM User
+ref: create-an-aws-iam-user
+content: |
+  Create a new programmatic IAM user to use in CSFLE-enabled clients.
+  The user will encrypt and decrypt the remote master key and must have
+  full ``List`` and ``Read`` permissions for the KMS service.
+
+  .. admonition:: Client IAM User Credentials
+     :class: note
+     
+     The CSFLE-enabled client takes the IAM User's :guilabel:`Access Key
+     ID` and :guilabel:`Secret Access Key` as configuration values. Note
+     these down for later when we reconfigure the client.
+---
+title: Create the Master Key
+ref: create-the-master-key
+content: |
+  In AWS KMS, generate a new master key. The key's name and description
+  don't affect the functionality of CSFLE but should describe that it's
+  for the CSFLE-enabled client.
+  
+  In the :guilabel:`Usage Permissions` step of the key generation
+  process, select the newly created IAM User with full KMS ``List`` and
+  ``Read`` permissions. This allows the user to encrypt and decrypt the
+  new master key.
+
+  .. important::
+     
+     The new client IAM User *should not* have administrative permissions
+     for the master key.
+---
+title: Specify the AWS KMS Provider Credentials
+ref: specify-the-aws-kms-provider-credentials
+content: |
+  Unlike the local KMS provider, the AWS KMS provider does not accept
+  the master key directly from the client configuration code. Instead,
+  it accepts the :guilabel:`Access Key ID` and :guilabel:`Secret Access
+  Key` of the IAM user with permission to encrypt and decrypt the master
+  key.
+
+  Update the KMS Provider configuration in CSFLE-enabled client creation code:
```

Comment:
I think I owe you the code for using AWS KMS, but I have not had the chance to build a working example yet. Do you know if this code works? I think the ARN might be necessary as it will identify the appropriate AWS zone.